server/src/main/java/org/elasticsearch/action/admin/cluster/snapshots/get/TransportGetSnapshotsAction.java

```diff
@@ -170,16 +181,17 @@ protected void masterOperation(final GetSnapshotsRequest request, final ClusterS
      *                            if false, they will throw an error
      * @return list of snapshots
      */
-    private List<SnapshotInfo> snapshots(@Nullable SnapshotsInProgress snapshotsInProgress, String repositoryName,
+    private List<SnapshotInfo> snapshots(SnapshotsInProgress snapshotsInProgress, String repositoryName,
                                          List<SnapshotId> snapshotIds, boolean ignoreUnavailable) {
```

Comment:
Nit: spurious whitespace deviation from master :)

```suggestion
                                        List<SnapshotId> snapshotIds, boolean ignoreUnavailable) {
```

Reply:
The version here is actually nicer/more correct :D that's why I deviated. I'd rather fix this in master next time a change to the file is made there :)